src/Library/demos/Breakpoints/main.js

```diff
@@ -0,0 +1,14 @@
+import Gtk from "gi://Gtk";
+
+const breakpoint = workbench.builder.get_object("breakpoint");
+const image = workbench.builder.get_object("image");
+
+breakpoint.connect("apply", () => {
+  image.icon_size = Gtk.IconSize.NORMAL;
```

Comment:
I feel like we might as well do this in blueprint too and only do the `console`.log in Code

wdtyt?